groovy/src/main/java/org/apache/zeppelin/groovy/GroovyInterpreter.java

```diff
@@ -55,159 +55,172 @@
 
 /**
  * Groovy interpreter for Zeppelin.
+ * @author dlukyanov@ukr.net / dmitry lukyanov
```

Comment:
@dlukyanov I see. Yeah so sadly Zeppelin doesn't encourage the footprint 😭 (see here: https://github.com/apache/zeppelin/commit/d64920d20b1d564faea431541a80dc876d35bd94)